src/gc/unix/cgroup.cpp

```diff
@@ -437,7 +437,18 @@ size_t GetRestrictedPhysicalMemoryLimit()
     size_t physical_memory_limit;
  
     if (!CGroup::GetPhysicalMemoryLimit(&physical_memory_limit))
```

Comment:
This might be neater using `||` instead of nested `if`s

Reply:
Or drop the `else`, like:

```
if (!CGroup::GetPhysicalMemoryLimit(&physical_memory_limit))
    return 0;

// If there's no memory limit specified on the container this 
// actually returns 0x7FFFFFFFFFFFF000 (2^63-1 rounded down to 
// 4k which is a common page size). So we know we are not
// running in a memory restricted environment.
if (physical_memory_limit > 0x7FFFFFFF00000000)
    return 0;
```

Reply:
done!